### ISSUE TITLE
AttributeError: module 'nngt' has no attribute 'plot'

### ISSUE THREAD

**Issue by SBottani** [Silmathoron/NNGT#144]
Hi again.,

In version 2.0.0  AttributeError: module `nngt' has no attribute 'plot`
nng.plot module is missing.

?

**Comment by Silmathoron** [comment]
Please include a complete minimal working example so that I can check what's wrong (plotting definitely works on my computer). Also 2.0.0 is pretty old now so you might want to upgrade to 2.2.0

**Comment by SBottani** [comment]
Of course sorry.
`run plot_chord_diagram.py`
 (the example file of the doc)
```

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~/Bureau/plot_chord_diagram.py in <module>
     59 sg = g.get_structure_graph()
     60 
---> 61 nngt.plot.chord_diagram(sg, names="name", sort="distance",
     62                         use_gradient=True, show=True)

AttributeError: module 'nngt' has no attribute 'plot'
```
Same if we run our other codes. On two different computers with recent installations.

**Comment by Silmathoron** [comment]
Do you have matplotlib installed?
Sorry but I really need a **complete minimal working example** that I can run locally to find the problem, otherwise there's no way to know what's going on.

**Comment by Silmathoron** [comment]
OK, so this was a missing submodule in the PyPi release.
This should be fixed in 2.2.1
